js/sum.js

```diff
@@ -0,0 +1,46 @@
+let addBtn = document.getElementById('add-btn');
+let newNum = document.getElementById('new-num');
+let numsContainer = document.getElementById('added-nums-container');
+let sumElem = document.getElementById('sum');
+let isOdd = false;
+
+addBtn.addEventListener('click', () => {
+    if (newNum.value === '') {
+        alert('Enter a number, please!');
+        return;
+    }
+
+    isOdd = !isOdd;
+    let currentElem = document.createElement('div');   
+    isOdd ?  currentElem.setAttribute('class', 'current-added-elem odd') : currentElem.setAttribute('class', 'current-added-elem');  
+    let addedNum = document.createElement('span');
+    let deleteBtn = document.createElement('button');
+    deleteBtn.style.all = 'unset';
+    deleteBtn.innerText = '(delete)';
+    deleteBtn.setAttribute('id', 'delete');
+    deleteBtn.setAttribute('class', 'delete-btn'); 
+    let textNode = document.createTextNode((+newNum.value).toFixed(2));
+    addedNum.appendChild(textNode);
+    currentElem.appendChild(addedNum)
+    currentElem.appendChild(deleteBtn);
+    numsContainer.appendChild(currentElem);
+    sumElem.innerText = (+sumElem.innerText + +addedNum.innerText).toFixed(2);
+
+    deleteBtn.addEventListener('click', (event) => {
```

Comment:
Adding an event listener for each newly created button can cause performance issues. If we have 1000 button we will have 1000 listeners with mostly duplicated logic. Can we optimize this?